packages/lodestar/src/chain/chain.ts

```diff
@@ -332,8 +332,8 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     this.forkChoice.addBlock(block.slot, blockRoot, block.parentRoot);
     // await this.applyForkChoiceRule();
     await this.updateDepositMerkleTree(newState);
-    // update metrics
-    this.metrics.currentSlot.set(block.slot);
+
+    this.metrics.currentSlot.inc()
```

Comment:
Thats wrong, there might be skip slot

Reply:
Doesn't matter if it's a skip slot we still increment

Reply:
yeah but current slot in metrics will be wrong.
Exmp currentSlot = 1, blockSlot = 3 -> currentSlot=2 instead of 3

Reply:
wait what ...?

Reply:
processBlock and increment slot by more than 1